app/joinEvent/attendeeDetails.tsx

```diff
@@ -20,6 +20,7 @@ const formSchema = z.object({
 
 export default function AttendeeDetails() {
   const { code } = useLocalSearchParams() as { code: string };
+  const { id } = useLocalSearchParams() as { id: string };
```

Comment:
This feels kind of strange, no? Both identifies the same event so I feel like we can get rid of one and later fetch the other.